toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/SceneView.kt

```diff
@@ -118,20 +118,20 @@ public fun SceneView(
     onViewpointChangedForBoundingGeometry: ((Viewpoint) -> Unit)? = null,
     graphicsOverlays: GraphicsOverlayCollection = rememberGraphicsOverlayCollection(),
     sceneViewProxy: SceneViewProxy? = null,
-    sceneViewInteractionOptions: SceneViewInteractionOptions = SceneViewInteractionOptions(),
-    viewLabelProperties: ViewLabelProperties = ViewLabelProperties(),
-    selectionProperties: SelectionProperties = SelectionProperties(),
+    sceneViewInteractionOptions: SceneViewInteractionOptions = remember { SceneViewInteractionOptions() },
+    viewLabelProperties: ViewLabelProperties = remember { ViewLabelProperties() },
+    selectionProperties: SelectionProperties = remember { SelectionProperties() },
     isAttributionBarVisible: Boolean = true,
     onAttributionTextChanged: ((String) -> Unit)? = null,
     onAttributionBarLayoutChanged: ((AttributionBarLayoutChangeEvent) -> Unit)? = null,
-    cameraController: CameraController = GlobeCameraController(),
+    cameraController: CameraController = remember { GlobeCameraController() },
     analysisOverlays: AnalysisOverlayCollection = rememberAnalysisOverlayCollection(),
     imageOverlays: ImageOverlayCollection = rememberImageOverlayCollection(),
     atmosphereEffect: AtmosphereEffect = AtmosphereEffect.HorizonOnly,
     timeExtent: TimeExtent? = null,
     onTimeExtentChanged: ((TimeExtent?) -> Unit)? = null,
     spaceEffect: SpaceEffect = SpaceEffect.Stars,
-    sunTime: Instant = Instant.parse("2000-09-22T12:00:00Z"),
+    sunTime: Instant = remember { Instant.parse("2000-09-22T12:00:00Z") },
```

Comment:
As above with PaddingValues, `Instant` is immutable. so we could define a global static property for it, something like:
```suggestion
    sunTime: Instant = SceneViewDefaults.DefaultSunTime
```
and elsewhere in MapView.kt:
```kotlin
object SceneViewDefaults {
    public val DefaultSunTime: Instant = Instant.parse("2000-09-22T12:00:00Z")
   ...
}
```